src/main/java/gov/nasa/jpl/mbee/mdk/mms/validation/ElementValidator.java

```diff
@@ -151,11 +149,12 @@ private void processServerElements(Map<String, Pair<Element, ObjectNode>> client
                     // solves edge case where first model validation incorrectly removes bins from project
                     removeServerObjectNodeUsingIdPrefix(elementObjects, MDKConstants.HOLDING_BIN_ID_PREFIX);
                 }
+                //Remove View Instances Bin
                 removeServerObjectNodeUsingIdPrefix(elementObjects, MDKConstants.VIEW_INSTANCES_BIN_PREFIX);
 
                 for(ObjectNode jsonObject : elementObjects) {
                     JsonNode idValue = jsonObject.get(MDKConstants.ID_KEY);
-                    if(idValue != null && idValue.isTextual() && !serverElementMap.containsKey(idValue.asText())) {
+                    if(idValue != null && idValue.isTextual() && !idValue.asText().startsWith(MDKConstants.HIDDEN_ID_PREFIX) && !serverElementMap.containsKey(idValue.asText())) {
```

Comment:
This check shouldn't be necessary. You've already filtered out the hidden elements above.

Reply:
Incorrect. The step above would only have removed the HIDDEN_ID's if a serverElementMap is passed to the run() command (I can't find a case where that happens).  It seems as though the method is expecting a serverElementMap to come from somewhere else. The HIDDEN step above is placed there in case of a leak from another step.